### PR TITLE
Finalized my blog post and hide details.

### PR DESCRIPTION
**Pull request by PalmPalm7**
1. Cut out project details to make it more generics
2. Populated original survey data with augumented data.
3. Reworded a few phrases to make it more smooth.

### CODE REVIEW

File: documentation/project-guides/presentation_principles.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+# 3 Principles
+1. Use data-driven strategies for storytellings. Support your conclusions and hypothesis with data.
+2. Know your audience, level your presentations with your clients and explain in ways they can understand.
+3. Embrace [Ownership Principle](https://www.amazon.jobs/content/en/our-workplace/leadership-principles). Task shouldn't be limited to completing the code/presentation/report; avoid the mindset that the rest is "not my job". Consider the bigger picture of how findings could benefit the client.
+
+# Case Study
+## Case Study Background
+
+The case study used in the blog post is from a Spark! data science project that centers around unanswered survey data, where we want to help analyze survey data over the past 8-9 years. For a desensitized data, please refer to [this spreadsheet](https://docs.google.com/spreadsheets/d/1wBBV9612tsHR8MsMihHI3gY7ebE2YMsE/edit?usp=sharing&ouid=101499547981540031808&rtpof=true&sd=true) (Every data entry is populated with augmented data).
```
Comment (mvoong18) [inline documentation/project-guides/presentation_principles.md:23]: I think giving background on the data is enough, remove the the link to the data

File: documentation/project-guides/presentation_principles.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+## Case Study Practice
+
+Before going deeper into the blog post, imagine you were given a tabular survey data for 20 survey responders across 10 years e.g. "Effectiveness of head coach's communication skills (on a scale from 1 to 4)". How would you use it to answer the following questions, "How have athletes' perceptions of coach communication evolved over the 8-9 year period?". Se the highlighted two rows below.
+
+> Survey Data
```
Comment (mvoong18) [inline documentation/project-guides/presentation_principles.md:29]: Also remove the screenshots of the survey data and head coach change

Comment (PalmPalm7) [reply documentation/project-guides/presentation_principles.md]: Hi Michelle, I have made changes according to your comments. Thank you for that!

File: documentation/project-guides/presentation_principles.md (hunk 1 above)
Comment (PalmPalm7) [reply documentation/project-guides/presentation_principles.md]: Hi Michelle, I have made changes according to your two comments. Thank you for that!

**Pull request merged**